src/Catty/Firebase/FirebaseAnalyticsReporter.swift

```diff
@@ -21,15 +21,16 @@
  */
 
 import Firebase
-@testable import Pocket_Code
 
-class AppDelegateMock: AppDelegate {
+class FirebaseAnalyticsReporter {
+    @objc open var analytics: Analytics.Type
 
-    var crashlyticsMock: Crashlytics
-
-    override open var crashlytics: Crashlytics { crashlyticsMock }
+    @objc init(analytics: Analytics.Type) {
+        self.analytics = analytics
+        setupAnalytics()
+    }
 
-    init(crashlytics: Crashlytics) {
-        self.crashlyticsMock = crashlytics
+    func setupAnalytics() {
```

Comment:
Please do not make this function publicly available. Set access control to either `private` or `fileprivate`.